packages/diffhtml/rollup.main.config.js

```diff
@@ -1,6 +1,6 @@
 import babel from 'rollup-plugin-babel';
 import nodeResolve from 'rollup-plugin-node-resolve';
-import replace from 'rollup-plugin-replace';
+import replace from '@rollup/plugin-replace';
```

Comment:
The old rollup-plugin-replace did not work for some reason, upgrading seems to fix the issue.